src/main/java/com/synergy/backend/domain/booth/entity/BoothParticipation.java

```diff
@@ -26,7 +26,7 @@ public class AttendeeBooth {
     @JoinColumn(name = "attendee_id", nullable = false)
     private Attendee attendee;
 
-    public AttendeeBooth(Booth booth, Attendee attendee) {
+    public BoothParticipation(Booth booth, Attendee attendee) {
```

Comment:
생성자는 private으로 접근제한을 두시고 of 같은 static 메서드로만 생성할 수 있게하는 것이 좋을 것 같습니다.
(new BoothParticipatioon() 쓰지 못하도록)

```suggestion
    @Builder
    private BoothParticipation(Booth booth, Attendee attendee) {
```

Reply:
수정했습니다!